src/main/java/com/spotinst/sdkjava/model/converters/admin/messageCenter/MessageCenterConverter.java

```diff
@@ -0,0 +1,207 @@
+            }
+            if (createMessage.isContentSet()) {
+                apiCreateMessage.setContent(createMessage.getContent());
+            }
+            if (createMessage.isStickyUntilSet()) {
+                apiCreateMessage.setStickyUntil(createMessage.getStickyUntil());
+            }
+            if (createMessage.isTypeSet()) {
+                apiCreateMessage.setType(createMessage.getType());
+            }
+            if (createMessage.isMainIconSet()) {
+                apiCreateMessage.setMainIcon(createMessage.getMainIcon());
+            }
+            if (createMessage.isHyperlinkSet()) {
+                apiCreateMessage.setHyperlink(createMessage.getHyperlink());
+            }
+            if (createMessage.isSecondaryIconSet()) {
+                apiCreateMessage.setSecondaryIcon(createMessage.getSecondaryIcon());
+            }
+            if (createMessage.isTimestampSet()) {
+                apiCreateMessage.setTimestamp(createMessage.getTimestamp());
+            }
+            if (createMessage.isAudienceSpecificationSet()) {
+                apiCreateMessage.setAudienceSpecification(createMessage.getAudienceSpecification());
+            }
+            if (createMessage.isOrganizationAudienceSpecificationSet()) {
+                apiCreateMessage.setOrganizationAudienceSpecification(createMessage.getOrganizationAudienceSpecification());
+            }
+            if (createMessage.isStarredSet()) {
+                apiCreateMessage.setStarred(createMessage.getStarred());
+            }
+            if (createMessage.isMarkedReadAtSet()) {
+                apiCreateMessage.setMarkedReadAt(createMessage.getMarkedReadAt());
+            }
+            if (createMessage.isUserAudienceSpecificationSet()) {
+                List<ApiUsersAudienceSpecification> apiUsersAudienceSpecification = createMessage.getUserAudienceSpecification()
+                        .stream().map(MessageCenterConverter::toDal).collect(Collectors.toList());
+                apiCreateMessage.setUserAudienceSpecification(apiUsersAudienceSpecification);
+            }
+        }
+
+        return apiCreateMessage;
+    }
+
+    private static ApiUsersAudienceSpecification toDal(UserAudienceSpecification userOrganizationMapping) {
+        ApiUsersAudienceSpecification apiUsersAudienceSpecification = null;
+
+        if (userOrganizationMapping != null) {
+            apiUsersAudienceSpecification = new ApiUsersAudienceSpecification();
+
+            if (userOrganizationMapping.isOrganizationIdSet()) {
+                apiUsersAudienceSpecification.setOrganizationId(userOrganizationMapping.getOrganizationId());
+            }
+            if (userOrganizationMapping.isUserIdsSet()) {
+                apiUsersAudienceSpecification.setUserIds(userOrganizationMapping.getUserIds());
+            }
+
+        }
+
+        return apiUsersAudienceSpecification;
+    }
+
+    public static GetMessage toBl(ApiGetMessage apiGetMessage) {
```

Comment:
All fields defined in GetMessage class are not converted here. Please confirm which is the correct set of parameters in this object.

Reply:
Removed unused parameters